dynawo/sources/Solvers/Common/DYNSolver.h

```diff
@@ -46,7 +46,8 @@ typedef enum {
   NoChange = 0x00,
   ModeChange = 0x01,
   ZChange = 0x02,
```

Comment:
zchange to NotSilentZChange for clarity.

Reply:
Solved